### ISSUE TITLE
Change gray button color

### ISSUE THREAD

**Issue by caitlynmayers** [caitlynmayers/dukes#148]
### Location
http://test-dukes.pantheonsite.io/?s=recipe
http://test-dukes.pantheonsite.io/product/real-mayonnaise/

### Description
Please change the disabled or gray button colors to #767676 for AA conformance.

**Comment by caitlynmayers** [comment]
Fix verified.

**Comment by caitlynmayers** [comment]
@hessdev did this revert? it's failing now.

**Comment by caitlynmayers** [comment]
Fix verified.
